Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp-01-code-0001-previous.swift

```diff
@@ -4,7 +4,7 @@ import SwiftUI
 @Reducer
 struct SyncUpDetail {
   @ObservableState
-  struct State {
+  struct State: Equatable {
```

Comment:
at the beginning of the tutorial we already set the state that we will be marking all `State`s as equatable for testing, and so we should just be doing that for all new features.